signing/secp256k1/api/src/main/java/tech/pegasys/signers/secp256k1/api/SingleSignerProvider.java

```diff
@@ -12,15 +12,16 @@
  */
 package tech.pegasys.signers.secp256k1.api;
 
+import java.util.Arrays;
 import java.util.Collections;
 import java.util.Optional;
 import java.util.Set;
 
-public class SingleTransactionSignerProvider implements TransactionSignerProvider {
+public class SingleSignerProvider implements SignerProvider {
 
-  private final TransactionSigner signer;
+  private final Signer signer;
 
-  public SingleTransactionSignerProvider(final TransactionSigner signer) {
+  public SingleSignerProvider(final Signer signer) {
     if (signer == null) {
       throw new IllegalArgumentException(
           "SingleTransactionSignerFactory requires a non-null TransactionSigner");
```

Comment:
exception message still referencing to "TransactionSigner". (might perform a full text search)

Reply:
fixed.